CMakeLists.txt

```diff
@@ -18,12 +18,36 @@ add_definitions(-DQT_DISABLE_DEPRECATED_BEFORE=0)
 # custom openscap parameters.
 if (NOT DEFINED OPENSCAP_LIBRARIES OR NOT DEFINED OPENSCAP_INCLUDE_DIRS)
     include("${CMAKE_ROOT}/Modules/FindPkgConfig.cmake")
-    pkg_check_modules(OPENSCAP REQUIRED libopenscap>=1.2.11)
+    pkg_check_modules(OPENSCAP REQUIRED libopenscap>=1.2.0)
```

Comment:
this looks suspect

Reply:
I think this is legit: https://github.com/OpenSCAP/scap-workbench/commit/a2265f5a44bef600807ca64358cb68d24ceea09e

Reply:
yeah, it is